synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -160,4 +191,9 @@ Dialog_Guide::init_widget_values()
 		angle_widget->set_value(synfig::Angle::deg(curr_guide->angle).get());
 	else
 		angle_widget->set_value(curr_guide->angle.get());
+
+	synfig::RendDesc &rend_desc(canvas->rend_desc());
+	factor = std::fabs((rend_desc.get_w()/2)/rend_desc.get_tl()[0]);
```

Comment:
I know it is strange, but we may have different factors for X- and Y- axes.
So please compute the factors for both directions.

Btw, the computation may be wrong as you wrote. You are assuming everything is symmetric (and it usually is), but it may not be. It should be how many pixels it has in a direction divided by how many synfig unities it has in the same direction (Right - Left coordinates).

Reply:
totally forgot about the y part ill make the changes